src/index.jsx

```diff
@@ -36,53 +40,99 @@ export default class Script extends React.Component {
   constructor(props) {
     super(props);
     this.scriptLoaderId = `id${this.constructor.idCount++}`; // eslint-disable-line space-unary-ops, no-plusplus
+    this.pkID = `script-${shortid.generate()}`;
+    this.unmount = () => {
```

Comment:
Please make this a separate function in the component instead of creating it in constructor